packages/block-editor/package.json

```diff
@@ -28,6 +28,7 @@
 		"@wordpress/blocks": "file:../blocks",
 		"@wordpress/components": "file:../components",
 		"@wordpress/compose": "file:../compose",
+		"@wordpress/core-data": "file:../core-data",
```

Comment:
Why do we need this? This package shouldn't depend on core data?

Reply:
It comes from:

https://github.com/WordPress/gutenberg/blob/1c7794121a74fb8458e660f5c521c4915a69fffb/packages/block-editor/src/index.js#L5

Reply:
I don't see it used so it's probably copied over from `@wordpress/editor`.

Reply:
Fix ready: 8a017ee.